commands/hooks/yacc.go

```diff
@@ -296,3 +307,157 @@ func (command *YaccHookDisableCommand) DisableAction(context *cli.Context) error
 
 	return nil
 }
+
+// YaccHookSettingsCommand define the command to get setting for YACC hook
+type YaccHookSettingsCommand struct {
+	Settings *settings.BitAdminSettings
+	flags    *YaccHookGetSettingsCommandFlags
+}
+
+// YaccHookGetSettingsCommandFlags define the flags of the YaccHookSettingsCommand
+type YaccHookGetSettingsCommandFlags struct {
+	project    string
+	repository string
+}
+
+// GetCommand provide a ready to use cli.Command
+func (command *YaccHookSettingsCommand) GetCommand() cli.Command {
+	return cli.Command{
+		Name:   "get-settings",
+		Usage:  "Get settings for Yet Another Commit Checker hook",
+		Action: command.GetSettings,
+		Flags: []cli.Flag{
+			cli.StringFlag{
+				Name:        "project",
+				Usage:       "The `<project_key>` where the repository will be created",
+				Destination: &command.flags.project,
+			},
+			cli.StringFlag{
+				Name:        "repository",
+				Usage:       "The `<repository_name>` to create",
+				Destination: &command.flags.repository,
+			},
+		},
+		BashComplete: func(c *cli.Context) {
+			helper.AutoComplete(c, command.Settings.GetFileCache())
+		},
+	}
+}
+
+// GetSettings contains logic to get current yacc settings
+func (command *YaccHookSettingsCommand) GetSettings(context *cli.Context) error {
+	client, err := command.Settings.GetAPIClient()
+	if err != nil {
+		return err
+	}
+
+	if len(command.flags.project) <= 0 {
+		return errors.New("--project flag is required")
+	}
+	if len(command.flags.repository) <= 0 {
+		return errors.New("--repository flag is required")
+	}
+
+
+	yacc, err := GetHookSettings(
+		command.flags.project,
+		command.flags.repository,
+		yaccHookKey,
+		client,
+	)
+	if err != nil {
+		return err
+	}
+
+	data, err := json.Marshal(yacc)
+	if err != nil {
+		return err
+	}
+
+	isCorrect  := "NO"
+	isOK, diff := isYACCSettingsCorrectWithDiff(yacc)
+
+	if isOK == true {
+		isCorrect = "YES"
+	}
+
+	fmt.Printf("%s# %s# %s", isCorrect, data, diff.String())
+
+	return nil
+}
+
+// GetHookSettings return the YACC hook settings from bitbucket
+func GetHookSettings(projectKey, repositorySlug string, hookKey string, bc *bitclient.BitClient) (YaccSettings, error) {
```

Comment:
Can this get moved on the bitclient to stay consistent ? This DoGet wasn't meant to be public.

Reply:
Sure,  I was lazy :). Will update it.

Reply:
Created PR on bitclient.

Reply:
Merged  :)